### DISCUSSION TITLE
RSS feed using Atom 2.0

### DISCUSSION THREAD

**Discussion by abd3lraouf** [varabyte/kobweb#467] [Q&A] [answered]
I wonder if we can create Rss feed we can do it using some gradle task. But maybe we can create an xml page just like html page

For example localhost/rss.xml

**Comment by bitspittle** [comment, marked as answer]
Yes you can accomplish this using gradle!

Check out https://github.com/varabyte/kobweb?tab=readme-ov-file#generating-site-code-at-compile-time and in your case you'd want to write a file to `public/rss.xml`. and instead of `kotlin.srcDir(generateCodeTask)` you would use `kotlin.resourceDir(...)` instead.

**Reply by abd3lraouf** [reply]
Thank you so much @bitspittle really love it

**Reply by bitspittle** [reply]
You're very welcome.

I just added some docs as well for clarity: https://github.com/varabyte/kobweb#generating-resources (actually I think my advice here was slightly off, it should be `resources.srcDir`, not `kotlin.resourceDir`, I misremembered).

**Reply by bitspittle** [reply]
Also FYI we're going to investigate a revamp to resources in a near future release, which may make this even a little easier. If we can pull it off, you'll see it mentioned in the release notes. (The change should be backwards compatible so you can move forward with this approach for now)

**Reply by abd3lraouf** [reply]
ooh wow, thank you @bitspittle I will implement it as you said. It should not differ so much than the one I've added for creating the MardownData object.

**Reply by bitspittle** [reply]
Good luck! Let me know if you run into any issues.
